
float64 	timestamp
int32 		sensor_id
int32   	len
Obstacle[]   	obstacleVec
